pkg/platform/defaults.go

```diff
@@ -226,6 +226,15 @@ func setPlatformDefaults(p *v1.IntegrationPlatform, verbose bool) error {
 			Duration: 5 * time.Minute,
 		}
 	}
+
+	if p.Status.Build.MaxRunningBuilds <= 0 {
+		if p.Status.Build.BuildStrategy == v1.BuildStrategyRoutine {
+			p.Status.Build.MaxRunningBuilds = 3
```

Comment:
I think we should provide these defaults in the pkg/util/defautls/defaults.go resource so it's easier to inspect as it happens for other operator constants.

Reply:
I had a second look, I think this approach is fine, as we already have several defaults for the platform there. Sorry, for the noise :)